project-reports/2022/2022-Q4-Hyperledger-Ursa.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: 2022 Q4 Hyperledger Ursa
+parent: 2022
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# 2022 Q4 Hyperledger Ursa
+
+Created by Brent Zundel
+
+# Project Health
+
+Ursa was quiet this quarter. 
+
+As noted in the previous report, we hope to continue our recruitment of
+new maintainers and contributors.
+
+At the Ursa meetings, conversation is focused on a roadmap for the
+projects.
+
+# Required Information
+
+1.  <span style="color: rgb(68,68,68);"> <a href="https://wiki.hyperledger.org/display/TSC/Projects+have+two+quarters+to+comply+with+common+repo+structure?focusedCommentId=41591637#comment-41591637" rel="nofollow">Have you switched from master to main in all your
+repos</a> </span> <span style="letter-spacing: 0.0px;">? Yes</span>
+2.  <span class="placeholder-inline-tasks" style="color: rgb(23,43,77);text-decoration: none;"> <span style="color: rgb(68,68,68);">
+<a href="https://tsc.hyperledger.org/repository-structure.html" class="external-link" rel="nofollow">Have you implemented the Common
+Repository Structure in all your repos</a> </span> </span> <span style="color: rgb(23,43,77);text-decoration: none;">? Yes</span>
+3.  <span style="color: rgb(23,43,77);text-decoration: none;"> <span style="color: rgb(23,43,77);">Has your project implemented these
+inclusive language changes listed below to your repo? No </span></span>
+4.  <span style="color: rgb(23,43,77);text-decoration: none;"> <span style="color: rgb(23,43,77);">Have you added an <a href="https://wiki.hyperledger.org/display/TSC/Inclusive+Language+Example" rel="nofollow">Inclusive Language Statement</a> to your project's
+documentation and/or Wiki pages? No</span> </span>
+
+# Questions/Issues for the TSC
+
+There are no issues at this time.
+
+# Releases
+
+No releases have been made this quarter.
+
+# Overall Activity in the Past Quarter
+
+There was very little activity this past quarter as this
+<a href="https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Fursa/dashboard;subTab=technical?time=%7B%22from%22:%222022-10-01T06:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222022-12-31T07:00:00.000Z%22%7D" rel="nofollow">Ursa Activity Dashboard</a> shows. 
+
+# Current Plans
+
+Near the end of the quarter we began to see additional engagement from some
+possible new maintainers. We also reinvigorated the discussion about
+refactoring Ursa and began to make concrete plans to that end.
+
+# Maintainer Diversity
```

Comment:
Are these maintainers active? Or do they need to be moved to Emeritus status?

Reply:
we have recently updated the set of maintainers. Those listed there are valid for Q4, we have more now.